skorecard/reporting/report.py

```diff
@@ -17,7 +17,7 @@ def build_bucket_table(
     column: str,
     bucketer=None,
     bucket_mapping: Optional[BucketMapping] = None,
-    epsilon=0.00001,
+    epsilon=1,
```

Comment:
Note that the numerator and denominator of the WoE formula are decimals: changing epsilon to 1 will mess up heavily the WoE calculation. The very small number is there to ensure that there is no division by zero.

Reply:
As pointed in the comment below, epsilon=1 makes the calculations compatible with the WoEEncoder transformer.